src/MatrixRoomHandler.ts

```diff
@@ -25,9 +25,32 @@ const log = Logging.get("MatrixRoomHandler");
 
 const ACCOUNT_LOCK_MS = 1000;
 const EVENT_MAPPING_SIZE = 16384;
+const XEP0106_ESCAPING_TRANSFORMATIONS = {
+   "\"": "\22",
+   "&": "\26",
+   "'": "\27",
+   "/": "\2f",
+   ":": "\3a",
+   "<": "\3c",
+   ">": "\3e",
+   "@": "\40"
+}
+function XEP0106_APPLY_ESCAPING_TRANSFORMATIONS(roomId: string): string {
```

Comment:
Add a space to separate, and camelcase the function name please. Also, make it an exported function to test it easier.

Reply:
I think by `Add a space to separate` this you meant to add a new line after the object.